dev/tasks/dev/utils/inquiries.js

```diff
@@ -68,10 +68,21 @@ module.exports = {
 		return new Promise( ( resolve ) => {
 			inquirer.prompt( [ {
 				name: 'description',
-				message: 'Package description (one sentence):'
+				message: 'Package description (one sentence, must end with period):'
 			} ], ( answers ) => {
-				resolve( answers.description || '' );
+				resolve( appendPeriodIfMissing( answers.description ) );
 			} );
 		} );
 	}
 };
+
+function appendPeriodIfMissing( text ) {
+	text = text ? String.prototype.trim.call( text ) : '';
```

Comment:
This might be paranoid approach, but if text is `Number`, it will throw SyntaxError because `Number` has no `trim()` method. With the solution above number will be cast to string.